src/cryptography/fernet.py

```diff
@@ -134,6 +141,20 @@ def __init__(self, fernets):
     def encrypt(self, msg):
         return self._fernets[0].encrypt(msg)
 
+    def rotate(self, msg):
+        timestamp, data = self._fernets[0]._get_unverified_token_data(msg)
```

Comment:
Since this is a static method we can just call `Fernet._get_unverified_token_data`